### PR TITLE
tlt-2754 - manage people - remove shopper role

### PR DESCRIPTION
**Pull request by elliottyates**
This PR relates to the updated Fall 2016-2017 course selection flow; shoppers and prospective enrollees are no longer available in the manage people tool.

Other enhancements and fixes:
- audit log file enable
- Canvas role labels now come from Canvas
- verification of school allowed roles on remove

### CODE REVIEW

File: canvas_manage_course/settings/base.py [hunk 1]
```diff
@@ -238,7 +238,13 @@
             'level': _DEFAULT_LOG_LEVEL,
             'formatter': 'verbose',
             'filename': os.path.join(_LOG_ROOT, 'django-canvas_manage_course.log'),
-        }
+        },
+        'manage_people_audit_log_file': {
```
Comment (bermudezjd) [inline canvas_manage_course/settings/base.py]: Do you need a separate file for the audit app?

File: canvas_manage_course/settings/base.py [hunk 2]
```diff
@@ -251,6 +257,16 @@
             'level': _DEFAULT_LOG_LEVEL,
             'propagate': False,
         },
+        'manage_people': {
+            'handlers': ['default'],
+            'level': _DEFAULT_LOG_LEVEL,
+            'propagate': False,
+        },
+        'manage_people_audit_log': {
+            'handlers': ['manage_people_audit_log_file'],
```
Comment (bermudezjd) [inline canvas_manage_course/settings/base.py]: Can this use the `default` handler?

File: canvas_manage_course/settings/base.py [hunk 3]
```diff
@@ -251,6 +257,16 @@
...
+            'level': 'DEBUG',
```
Comment (bermudezjd) [inline canvas_manage_course/settings/base.py]: Does this need to be pinned to `DEBUG` logging?

File: manage_people/tests/test_views.py [hunk 4]
```diff
@@ -257,17 +277,17 @@ def test_get_enrollments_added_through_tool(self, mock_get_all_list_data):
...
         """ Ensure that registrar-fed enrollments aren't returned. """
         # set up the mock data
         user_id = uuid.uuid4().hex
         course_instance_id = uuid.uuid4().int
         canvas_enrollments = [  # result of enrollments.list_enrollments_sections
             {'user': {'sis_user_id': user_id, 'sortable_name': 'Burn, Acid'},
-             'role': 'Guest'},
-        ]
-        course_enrollees = [  # result of query for non-feed enrollments
+             'role': 'Guest', 'role_id': 9},
```
Comment (eparker71) [inline manage_people/tests/test_views.py]: I see the '}' after the 9, but I don't see the '{' to start. this might be extra or missing.

**Comment by eparker71** [comment]
@elliottyates this looks good, nothing is jumping out at me in the code. I know the code was not easy to navigate. I checked out the branch ran all the tests and ran the tool locally, it all worked. Good Job! 👍

**Comment by elliottyates** [comment]
Thanks @bermudezjd for the suggestions. Thanks @eparker71 for the quick turnaround and for testing the app locally.

**Pull request merged**